docs/product-spec.md

```diff
@@ -0,0 +1,138 @@
+# VxMail Product Spec
+
+## Background & Motivation
+
+The COVID-19 pandemic presents a new and significant challenge to the 2020
+General Election in the United States. How can we safely hold elections when it may
+be unsafe to congregate in a typical precinct? There is an emerging
+consensus that one important solution will be the expansion of
+vote-by-mail.
+
+## Key Hypothesis
+
+Today, many small and some medium-sized counties have a sufficiently
+manual process for a relatively small number of vote-by-mail
+ballots. COVID-19 will require those counties to process 10 times more
+ballots, making the manual process no longer practical—because of
+insufficient speed and accuracy. With simple, readily available
+off-the-shelf hardware and robust software, we can enable 3 election
+staffers to handle 50,000 vote-by-mail ballots assuming at least 4
+weeks for sending out ballots prior to election day and 2 weeks for
+tabulation.
+
+
+## VxMail
+
+We propose VxMail, a set of tools to help small counties quickly scale
+up their vote-by-mail capacity.
+
+### Components
+
+VxMail includes:
+* a Ballot Manager to send mail ballots to voters, log receipt of those ballots, and verify signatures.
+* tools to efficiently open envelopes and extract ballots.
+* a Ballot Tabulator to scan and tabulate ballots.
+
+The Ballot Manager works online for connecting with print-and-mail
+services, and potentially to the state's Election Management
+System. The Ballot Scanner is offline for security purposes.
+
+### Printing and mailing ballots
+
+The VxMail Ballot Manager lets an election official:
+
+* upload the election definition
+* upload the voter mailing list
+* proof generated ballots on screen
+* order those ballots printed and mailed, with a 3-day in-USPS-hands SLA
+* track ballot delivery to voters.
```

Comment:
```suggestion
* track ballot delivery to voters
```

It's a little odd that only the last one (or two?) have periods at the end. I'd go either all or nothing.